apps/web/src/components/EmailVerification/EmailVerificationDialog.tsx

```diff
@@ -171,6 +170,16 @@ export const EmailVerificationDialog = ({
     }
   }, [emailAddressContacted, requestACodeMessage, watchEmailAddressInput]);
 
+  /*  Populate the initial email address in the form.
+   *  This can change if the email address is updated and the dialog is reopened.
+   *  Therefore, it is a useEffect instead of using ReactHookForms default values.
+   */
+  useEffect(() => {
+    if (initialEmailAddress) {
+      requestACodeFormMethods.setValue("emailAddress", initialEmailAddress);
+    }
+  }, [initialEmailAddress, requestACodeFormMethods]);
+
```

Comment:
Okay this works on the initial dialog open. However, closing the dialog, then opening again, causes it to disappear.

I'm looking into it why now.